libs/core/src/lib/carousel/carousel.component.ts

```diff
@@ -373,13 +373,35 @@ export class CarouselComponent
     onKeydownArrowRight(event: KeyboardEvent): void {
         event.preventDefault();
         this._isRtl() ? this.previous() : this.next();
+        this.carouselContainer.nativeElement.focus();
     }
 
     /** @hidden */
     @HostListener('keydown.arrowleft', ['$event'])
     onKeydownArrowLeft(event: KeyboardEvent): void {
```

Comment:
Maybe combine these methods into one that defines which action to take based on rtl and vertical input?

Reply:
Yeap, makes sense